### PR TITLE
Update vuex-store.md

### PR DESCRIPTION
**Pull request by geoidesic**
Clarification for Dummies like me. Explains how to add vuex to an existing project.

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [x] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch and _not_ the `master` branch
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] It's been tested on a Cordova (iOS, Android) app
- [x] It's been tested on a Electron app
- [x] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs) (for faster update click on "Suggest an edit on GitHub" at bottom of page) or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [x] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: docs/src/pages/quasar-cli/vuex-store.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ We won't go into details on how to configure or use Vuex since it has great docs
 
 By default, if you choose to use Vuex when you create a project folder with Quasar CLI, it will set you up on using Vuex modules. Each sub-folder of `/src/store` represents a Vuex Module.
 
-If by contrast, you did not choose to use Vuex when you create a project folder with Quasar CLI, Adding a Vuex Module (see below) is all you need to do to `install` vuex.
+If by contrast you did not choose Vuex option during the project creation, adding a Vuex Module (see below) is all you need to install Vuex.
```
Comment (geoidesic) [inline docs/src/pages/quasar-cli/vuex-store.md:22]: I would suggest
"If by contrast you did not choose the Vuex option during project creation, adding a Vuex Module (see below) is all you need in order to install Vuex."

Comment (IlCallo) [reply docs/src/pages/quasar-cli/vuex-store.md]: The "by contrast" sounded strange, had a native speaker help me to rephrase it.
"in order to"/"to do to ..." etc are most of the time just superfluous words which don't carry semantic meaning, that's why I removed them.
In general concise phrases are easier to understand

**Comment by IlCallo** [comment]
@geoidesic thanks for bringing this up and holding your point :)

File: docs/src/pages/quasar-cli/vuex-store.md (hunk 1 above)
Comment (IlCallo) [reply docs/src/pages/quasar-cli/vuex-store.md]: For more references on the topic: https://v3.vuejs.org/guide/contributing/writing-guide.html#principles

**Pull request merged**